src/main.rs

```diff
@@ -332,28 +328,23 @@ fn main() {
         .unwrap()
         .map(|r| r.unwrap())
         .collect::<Vec<GameRow>>()
-    })();
-
-    rayon::scope_fifo(|scope| {
-        for game in gamerows.iter() {
-            let conn_arc = Arc::clone(&conn_mtx);
-            scope.spawn_fifo(move |_| {
-                println!("// Processing game #{}", game.id);
-                handle_game(&game, max_depth, plies_to_undo, !multi_tinue).and_then(|r| {
-                    if test {
-                        return None;
-                    }
+    };
 
-                    let local_conn = conn_arc.lock().unwrap();
-                    Some(local_conn.execute("INSERT INTO tinues(gameid, size, plies_to_undo, tinue_depth, tinue) VALUES(?, ?, ?, ?, ?)", 
-                        params![
-                            r.gameid,
-                            r.size,
-                            r.plies_to_undo,
-                            r.tinue_depth,
-                            r.tinue]).unwrap())
-                });
-            });
+    let conn_mtx: Mutex<Connection> = Mutex::new(conn);
+
+    gamerows.par_iter().for_each(|game| {
```

Comment:
The advantage I see of using the `_fifo` methods is that I can stop the program and start it to continue at roughly the same point.

But if you don't need the `Arc` here, then I probably don't need it in my code either. Unless `rayon` is doing some magic here that it doesn't if one spawns the tasks manually.